### DISCUSSION TITLE
Show help menu when no command is entered

### DISCUSSION THREAD

**Discussion by Jeff-RavenLabs** [funbiscuit/embedded-cli#18] [General]
HI,

Is there any way to configure the CLI to show the help menu when no command is submitted?

Thanks,
Jeff

**Comment by funbiscuit** [comment]
Hi!
No, currently help is shown only on "help" command. If no command is entered, it just does nothing and prints another invitation.
Do you thing another binding like "onEmptyCommand" should be configurable?

**Reply by Jeff-RavenLabs** [reply]
"Do you thing another binding like "onEmptyCommand" should be configurable?" - yes, I think this would be a great feature

**Reply by funbiscuit** [reply]
I've created a new issue #19, hopefully it will be added soon

**Reply by Jeff-RavenLabs** [reply]
Thank you!
